boards/arm/gd32f450i_eval/gd32f450i_eval-pinctrl.dtsi

```diff
@@ -30,4 +30,13 @@
 			drive-open-drain;
 		};
 	};
+
+	spi5_default: spi5_default {
+		group1 {
+			pinmux = <SPI5_SCK_PG13>, <SPI5_MOSI_PG14>,
+				 <SPI5_MISO_PG12>,
+				 /* Use pinmux to pullup pg10 and pg11. */
+				 <SPI5_IO2_PG10>, <SPI5_IO3_PG11>;
```

Comment:
nit: pull-up is not being enabled, does this mean SPI peripheral keeps these lines pulled-up when selected here?

Reply:
Yes.

If PG10 and PG11 configured as IO2 and IO3, it default behavior is output high level.

Better solution is use the WP# and HOLD# pins instead, but there doesn't have related logic in spi_nor flash driver.